ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/advanced-gas-fee-input-subtext.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Box from '../../../ui/box';
+import I18nValue from '../../../ui/i18n-value';
+
+const AdvancedGasFeeInputSubtext = ({ latest, historical }) => {
+  return (
+    <Box className="advanced-gas-fee-input-subtext">
+      <Box display="flex" alignItems="center">
+        <span className="advanced-gas-fee-input-subtext__label">
+          <I18nValue messageKey="currentTitle" />
+        </span>
+        <span>{latest}</span>
+        <img src="./images/high-arrow.svg" alt="" />
```

Comment:
Do we ever want this to be 'low-arrow.svg'? What will be the basis for selecting one versus the other?

Reply:
Ah I see now you mention in the PR description this along with some other values are hardcoded til the data is available.